.gitlab-ci.yml

```diff
@@ -110,8 +98,8 @@ Docker build git tag:
   script:
     - BRANCH_NAME=$(git branch -a --contains tags/${CI_COMMIT_TAG} | grep origin | sed 's/.*origin\///' | tail -1)
     - CLEAN_BRANCH_NAME=${BRANCH_NAME/\//-}
-    - docker pull $DOCKER_RELEASE_REGISTRY/ncigdc/$CI_PROJECT_NAME:${CLEAN_BRANCH_NAME}
-    - docker tag $DOCKER_RELEASE_REGISTRY/ncigdc/$CI_PROJECT_NAME:${CLEAN_BRANCH_NAME} $DOCKER_RELEASE_REGISTRY/ncigdc/$CI_PROJECT_NAME:${CI_COMMIT_TAG}
+    - docker pull docker.osdc.io/ncigdc/$CI_PROJECT_NAME:${CLEAN_BRANCH_NAME}
```

Comment:
bad merge?
Or are we hardcoding the server now?

Reply:
Old one did not work, 
we have containers.osdc.io, dev-containers.osdc.io to push images for tag and branches, 
and could not get this work, @a-collet suggested to use this hard coded address.